subworkflows/local/vcf_impute_beagle5/tests/nextflow.config

```diff
@@ -0,0 +1,20 @@
+process {
+    withName: BEAGLE5_BEAGLE {
+        ext.prefix = { "${meta.id}_beagle5_imputed" }
+        ext.args = "ne=20000 nthreads=4 gp=true ap=true"
+    }
+    
+    withName: 'VCF_IMPUTE_BEAGLE5:BCFTOOLS_VIEW' {
```

Comment:
We will need to standardize the way the sbwf define the name and the process allocated in their respective config.
Could you open an issue on that ?